docs/usage/rook-ceph-install.md

```
@@ -0,0 +1,437 @@
+- LVM2 packages installed on all nodes
+- Minimum 2 GB RAM per node (4 GB+ recommended)
+- `helm` installed if using Helm-based deployment (optional)
+
+## Installation Steps
+
+### Step 1: Clone the Rook Repository
+
+*Rook is the unique Kubernetes operator for managing Ceph workloads, which is
+why cloud-storage uses Rook. The SAP cloud infrastructure is built on top of
+Kubernetes, and the automation and foundation stack is Kubernetes. Because
+Rook is the Kubernetes operator, SAP has chosen it for managing Ceph workloads. *
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)